common/djangoapps/util/password_policy_validators.py

```diff
@@ -256,7 +256,9 @@ def __init__(self, min_numeric=0):
         self.min_numeric = min_numeric
 
     def validate(self, password, user=None):
-        if _validate_condition(password, lambda c: c.isdigit(), self.min_numeric):
+        if not isinstance(password, unicode):
+            password = password.decode()
+        if _validate_condition(password, lambda c: c.isnumeric(), self.min_numeric):
```

Comment:
previously we changed the `isnumeric` function to `isdigit`. Why are we doing the opposite here?

Reply:
We were changing the `isnumeric()` function into `isdigit()` because `isnumeric()` only works on Unicode, and when we ran command `changepassword`, we got password as an `str` instead of Unicode. But, now, as we have added a check of Unicode and shifting the password manually to Unicode, that means `isnumeric()` will work.